packages/tupaia-web/src/features/EntitySearch/EntityMenu.tsx

```diff
@@ -100,12 +99,21 @@ const EntityMenuItem = ({
         <MenuLink to={link}>
           {entity.name} {entity.type === 'facility' && <HospitalIcon />}
         </MenuLink>
-        <IconButton onClick={onExpand} disabled={parentIsLoading || !nextChildren}>
+        <IconButton
+          onClick={toggleExpanded}
+          disabled={parentIsLoading || !nextChildren}
+          aria-controls="entity-expand-button"
```

Comment:
Nice, thank you! Can we just add an `aria-label` of something like `toggle menu for ....` ?